Doc/howto/descriptor.rst

```diff
@@ -934,32 +934,51 @@ here is a pure Python equivalent:
             if doc is None and fget is not None:
                 doc = fget.__doc__
             self.__doc__ = doc
+            self.name = None
```

Comment:
More convenient to make this:  `self._name = ''`

Reply:
Changed, thanks